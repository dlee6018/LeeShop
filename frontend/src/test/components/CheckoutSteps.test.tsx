import React from "react";
import CheckoutSteps from "../../components/CheckoutSteps";
import {shallow} from "enzyme"


it("renders CheckoutSteps component", () => {
    const component = shallow(<CheckoutSteps/>)
    expect(component).toMatchSnapshot()
})