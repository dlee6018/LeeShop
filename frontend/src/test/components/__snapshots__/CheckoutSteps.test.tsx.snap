// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CheckoutSteps component 1`] = `
<Nav
  className="justify-content-center mb-4"
  fill={false}
  justify={false}
>
  <NavItem>
    <NavLink
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      disabled={true}
    >
      Sign In
    </NavLink>
  </NavItem>
  <NavItem>
    <NavLink
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      disabled={true}
    >
      Shipping
    </NavLink>
  </NavItem>
  <NavItem>
    <NavLink
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      disabled={true}
    >
      Payment
    </NavLink>
  </NavItem>
  <NavItem>
    <NavLink
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      disabled={true}
    >
      Place Order
    </NavLink>
  </NavItem>
</Nav>
`;
